scripts/ssm-build.sh

```diff
@@ -197,3 +200,15 @@ fpm -s pleaserun -t "$PACK_TYPE" \
 --depends apel-ssm \
 --package "$BUILD_DIR" \
 /usr/bin/ssmreceive
+
+echo "Possible Issues to Fix:"
+if [ "$OS_EXTENSION" == "_all" ]
+then
+    # Check the resultant debs for 'lint'
+    TAG="$VERSION-$ITERATION"
+    lintian $BUILD_DIR/apel-ssm_${TAG}_all.deb
+    lintian $BUILD_DIR/apel-ssm-service_${TAG}_all.deb
```

Comment:
```suggestion
    lintian "$BUILD_DIR"/apel-ssm_"${TAG}"_all.deb
    lintian "$BUILD_DIR"/apel-ssm-service_"${TAG}"_all.deb
```

Ah, the code quality check picked this one up: https://codeclimate.com/github/apel/ssm/pull/323